src/emittermixin.js

```diff
@@ -357,6 +367,48 @@ const EmitterMixin = {
 				destinations.delete( emitter );
 			}
 		}
+	},
+
+	/**
+	 * Returns an `EmitterMixin` instance with given `id`, which this emitter is listening to. Returns `null` if
+	 * this emitter do not listen to any emitter identified by `id`.
+	 *
+	 * @protected
+	 * @param {String} id Unique emitter id.
+	 * @returns {EmitterMixin|null}
+	 */
+	_getEmitterListenedTo( id ) {
```

Comment:
Please do not add more methods to the emitter mixin *object*. Add them to the module. Simply export them from this module and replace `this` with `subject` parameter.

The more methods we add to the object, the bigger chance that it's going clash or something. This is a mixin which should safely work with any class. Besides, it's also uglier.